### PR TITLE
feat: integration with EPD Visualization (#13933)

### PR DESCRIPTION
**Pull request by sam-garland**
Commit for initial review.
Provides components and services for visual spare parts picking.

At least the following tasks remain:
- Increase unit test coverage
- Add e3e tests
- Add automated accessibility testing
- Update to handle 5 tabs nicely
- Add sample data
- Some UI updates
- Update UI5 version when 1.96.0 released

image

### CODE REVIEW

**Comment by sam-garland** [comment]
Will use `epic/epd-visualization` branch instead.

File: angular.json [hunk 1]
```diff
@@ -16,6 +16,11 @@
             "customWebpackConfig": {
               "path": "./extra-webpack.config.js"
             },
+            "optimization": {
```
Comment (wangzixi-diablo) [inline angular.json:19]: Do we introduce this setting due to this known issue? https://github.com/angular/angular-cli/issues/20864

Need to evaluate if any potential side effect.

Comment (sam-garland) [reply angular.json:19]: I will remove it. I was getting a build issue when I had inline SVG in my SCSS, but I have removed the inline content now.

**Comment by sam-garland** [comment]
I have reopened this PR against a different branch here:
https://github.com/SAP/spartacus/pull/14126

File: angular.json (hunk 1 above)
Comment (sam-garland) [reply angular.json:19]: @wangzixi-diablo I have removed this in #14126 in this commit: https://github.com/SAP/spartacus/pull/14126/commits/97c7168d905c7d1e96be55deedcd28f4e29c7023

**Pull request closed without merging**